test/lit/passes/gufa-cast-all.wast

```diff
@@ -146,3 +146,155 @@
     )
   )
 )
+
+;; Imported tags may be written to from places we do not see.
+(module
+  ;; CHECK:      (type $0 (func (param i32)))
+
+  ;; CHECK:      (type $1 (func))
+
+  ;; CHECK:      (import "fuzzing-support" "throw" (func $throw (type $0) (param i32)))
+  (import "fuzzing-support" "throw" (func $throw (param i32)))
+  ;; CHECK:      (import "fuzzing-support" "tag" (tag $tag (type $0) (param i32)))
+  (import "fuzzing-support" "tag" (tag $tag (param i32)))
+
+  ;; CHECK:      (export "func" (func $func))
+
+  ;; CHECK:      (func $func (type $1)
+  ;; CHECK-NEXT:  (try
+  ;; CHECK-NEXT:   (do
+  ;; CHECK-NEXT:    (call $throw
+  ;; CHECK-NEXT:     (i32.const 1)
+  ;; CHECK-NEXT:    )
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:   (catch $tag
+  ;; CHECK-NEXT:    (drop
+  ;; CHECK-NEXT:     (pop i32)
+  ;; CHECK-NEXT:    )
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT: )
+  (func $func (export "func")
+    (try
+      (do
+        (call $throw
+          (i32.const 1)
+        )
+      )
+      (catch $tag
+        ;; The i32 result of the tag will be stored to a local. If we thought no
```

Comment:
The result doesn't seem to have a local?

Reply:
Oh right, I guess this was before I simplified the test. Fixed, thanks.